addons/web_editor/models/ir_attachment.py

```diff
@@ -36,7 +36,7 @@ def _compute_image_src(self):
                 attachment.image_src = attachment.url
             else:
                 # Adding unique in URLs for cache-control
-                unique = attachment.checksum[:8]
+                unique = (attachment.checksum or '')[:8]
```

Comment:
so if unique is not defined, it should not be in the URL. You don't want a `foo.jpg?unique=` or `/we/image/42-/foo`